OTA-1814: docs(alerts): README updates for #2279 (#2296)
openshift/oc: pkg/cli/admin/upgrade/recommend/examples/README.md

# Examples for `oc adm upgrade recommend`

> Each testcase is anchored by an `xxx-cv.yaml`, where `xxx` is a common substring that identifies the testcase. The `-cv.yaml` file describes the ClusterVersion object essential to the functionality of `oc adm upgrade recommend`

Reviewer: ```suggestion
Each testcase is anchored by a `<test-case>-cv.yaml` file which describes the ClusterVersion object and is essential to the testing of `oc adm upgrade recommend`.
```

**INPUTS**

> ---

Reviewer: nit: We would not need a divider line here if we use `## INPUTS` in Line 5.

* `TESTCASE-cv.yaml`: ClusterVersion object (required, created by `oc get clusterversion version -o yaml`).  Lists are also supported.
* `TESTCASE-featuregate.yaml`: FeatureGate object (optional, created by `oc get featuregate cluster -o yaml`). Lists are NOT supported.
* `TESTCASE-infrastructure.yaml`: Infrastructure object (optional, created by `oc get infrastructure cluster -o yaml`). Lists are NOT supported

> * `TESTCASE-alerts.json`: Alerts currently present in the cluster (optional, created by `OC_ENABLE_CMD_INSPECT_ALERTS=true oc adm inspect-alerts`)

Reviewer: Interesting, Trevor did not use this command directly in https://github.com/openshift/oc/commit/26ec958977d4a63ae2ee37dae86a80c2b299cc2a
```suggestion
* `TESTCASE-alerts.json`: Running alerts in the cluster (optional, expected output of `OC_ENABLE_CMD_INSPECT_ALERTS=true oc adm inspect-alerts`)
```

The wording is from

```console
$ OC_ENABLE_CMD_INSPECT_ALERTS=true oc adm inspect-alerts -h
Collect information about running alerts.

Usage:
  oc adm inspect-alerts [flags] [options]

Use "oc options" for a list of global command-line options (applies to all commands).
```

Author: What change should be made here?
